### ISSUE TITLE
Tooltip for building supply quality stuck

### ISSUE THREAD

**Issue by Samuel38** [voxeltycoon/bug-tracker#205]
**Describe the bug**
After hovering over the tooltip of the supply quality of a city's building, the tooltip got stuck. 
Displaying it again on the same building wouldn't unstuck it. Nothing did but saving/reloading. 

**Steps to reproduce**
 - Open the city building menu. 
 - Hover over the supply quality level to display tooltip. 
 - I stayed some time, I have no idea what I did exactly, I was reading. 
 - At the end the tooltip would not disappear. I had to reload the save. 

**Build version**
0.85.0.24 steam

**Platform**
Windows 10

**Reproducibility**
Happened only once so far.

**Comment by polak-jan** [comment]
This really seems to be a rather rare edge case, haven't been able to confirm it myself from somewhat quick testing of various way to force it. Hopefully more people will encounter this as it would help diagnose it.

**Comment by Samuel38** [comment]
Yes, I haven't been able to reproduce it either so far. I'll definitely come back to you if I manage to do so.

**Comment by BastiJava** [comment]
I have the same bug more often. But i can't reproduce it either. It's really annoying. Only save/load can get the tooltip back.
